test/render/gltf-test-cases.js

```diff
@@ -0,0 +1,26 @@
+import {AnimationLoop} from '@luma.gl/core';
+
+const examples = {
+  avocado: require('../../examples/gltf/app').avocado,
+  milkTruck: require('../../examples/gltf/app').milkTruck
+};
+
+export default Object.keys(examples).map(name => {
+  const app = examples[name];
+  const animationLoop = new AnimationLoop(app);
+  let count = 0;
+  return {
+    name,
+    onInitialize: params => {
+      return animationLoop.onInitialize(params);
+    },
+    onRender: params => {
+      animationLoop.onRender(params);
+      if (count++ > 2000) {
+        params.done();
+      }
+    },
+    onFinalize: animationLoop.onFinalize.bind(animationLoop),
+    goldenImage: `./test/render/golden-images/${name}.png`
```

Comment:
The golden images should be in this PR

Reply:
The problem is I am only getting empty images, not sure why.